libs/community/tests/integration_tests/vectorstores/test_bigquery_vector_search.py

```diff
@@ -0,0 +1,88 @@
+"""Test BigQuery Vector Search.
+In order to run this test, you need to install Google Cloud BigQuery SDK
+pip install google-cloud-bigquery
+Your end-user credentials would be used to make the calls (make sure you've run
+`gcloud auth login` first).
+"""
+
+import uuid
+
+import pytest
+from google.cloud import bigquery
```

Comment:
I'm not sure this import should be on this level, since this library is not installed on github runners.

Reply:
Good point. This may be better within `store` and `teardown` methods.